src/test/librbd/cache/test_ReplicatedWriteLog.cc

```diff
@@ -0,0 +1,92 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "test/librbd/test_fixture.h"
+#include "test/librbd/test_support.h"
+#include "librbd/ImageCtx.h"
+
+void register_test_replicatedwritelog() {
+}
+
+static int get_features(bool *old_format, uint64_t *features)
```

Comment:
Nit: why are these common functions re-defined here?

Reply:
I removed this file as the test case is covered in test_librbd.cc.